webpack.dev.js

```diff
@@ -4,20 +4,24 @@ const merge = require('webpack-merge');
 const path = require('path');
 const common = require('./webpack.common');
 
+const filePath = path.join(__dirname, 'browser/dist');
+
 module.exports = merge(common, {
   mode: 'development',
   resolve: {
     extensions: ['.js'],
   },
   devtool: 'source-map',
   devServer: {
-    contentBase: path.join(__dirname, 'browser/dist'),
+    contentBase: filePath,
     historyApiFallback: true,
     hot: true,
     proxy: {
       '/': 'http://localhost:3000',
     },
     port: 3001,
+    // eslint-disable-next-line no-shadow
+    writeToDisk: filePath => /index\.html/.test(filePath),
```

Comment:
This dev-server feature writes `index.html` locally instead of in memory. This allows the express.js route catch-all to work correctly.